### DISCUSSION TITLE
bitr()id转换不唯一，该如何处理呢？

### DISCUSSION THREAD

**Discussion by JackChen2024** [YuLab-SMU/clusterProfiler#548] [Q&A]
老师您好！
在使用bitr()函数进行ensembl id与symbol之间的转换时，我遇到了这样一个问题：同一个id对应了多个symbol，并且其中一个symbol也对应有多个id。其中有一种对应方式是正确的。
举例如下：
![image](https://user-images.githubusercontent.com/126886139/222736266-b5f3ae16-9378-4c3d-bc5b-37dfa8681933.png)
![image](https://user-images.githubusercontent.com/126886139/222736819-038337aa-0ca6-4628-9e39-62d5b06cb50f.png)
（在本例中，除了图中所示的重复映射，另外两个对应关系是正确且唯一的）
由于我是一个本科阶段的初学者，我想请问一下这种现象是否是正常的呢？在数据处理中，我应该如何处理这种矛盾比较好呢？

**Comment by zhen-fu** [comment]
I have run into this before, I usually just ignore it. As there were not that many genes that with the symbol but different Ensembl IDs. 
Or, in the enrichment analysis, you can use Ensemble ID instead of symbols.
